WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -36,20 +42,34 @@ - (void)dealloc
     _tableViewHandler.delegate = nil;
 }
 
-- (instancetype)init
++ (CommentsViewController *)controllerWithBlog:(Blog *)blog
 {
-    self = [super init];
-    if (self) {
-        self.restorationClass = [self class];
-        self.restorationIdentifier = NSStringFromClass([self class]);
-    }
-    return self;
+    NSParameterAssert([blog isKindOfClass:[Blog class]]);
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"CommentsList" bundle:nil];
+    CommentsViewController *controller = [storyboard instantiateInitialViewController];
+    [controller setupWithBlog:blog];
+    return controller;
+}
+
+- (void)setupWithBlog:(Blog *)blog
+{
+    self.blog = blog;
+    self.restorationClass = [self class];
+    self.restorationIdentifier = NSStringFromClass([self class]);
```

Comment:
I want to say whenever we have a storyboard or a xib for a view controller we tend to set the restoration identifier via the identity panel. We could do the same here for consistency now that there's a storyboard.  Or not.  Up to you. :)
<img width="309" alt="Screen Shot 2021-02-24 at 9 54 19 AM" src="https://user-images.githubusercontent.com/1435271/109028614-978a0700-7687-11eb-8d72-152982fb7357.png">